lib/js/components/Icons/Icon/Icon.vue

```diff
@@ -9,6 +9,7 @@
 			[rotationClass]: rotationClass,
 			[sizeClassName]: true,
 		}"
+		:style="{'--rotate': `${rotation}deg`}"
```

Comment:
```suggestion
		:style="{'--rotate': `${rotation ?? 0}deg`}"
```
And then we could get rid of `rotationClass` computed and `-ds-rotate` from css (move `transform` to `.ds-icon`)